### ISSUE TITLE
Add Shuffling of Dataset while Training

### ISSUE THREAD

**Issue by meshde** [meshde/EruditeX#32]
Add feature to shuffle the input data after each epoch in `dtrnnTrain.py`

**Comment by mitrjain96** [comment]
Use sklearn.utils.shuffle
